src/earn/incentives.md

```diff
@@ -4,41 +4,67 @@ title: Project Artemis
+During the Cudos Public 'Somniorum' Testnet we will be running our incentive programme Project Artemis. We're inviting the Cudos community to help contribute towards the testing to help move the network closer to mainnet. To help maximise participation in our public testnet and to get the best feedback we're offering token rewards to users who join us and fulfil some key testing tasks.
 
-During Project Artemis participants will earn points by completing tasks. Once our mainnet is deployed participants of Project Artemis will have native CUDOS airdropped to their wallets as reward for their contributions.
+During Project Artemis participants are able to earn CUDOS by completing tasks. Once our mainnet is deployed participants of Project Artemis will have native CUDOS airdropped to their wallets as reward for their contributions.
 
-As the public testing phases progress we'll be posting more detail on rewards and community progress here.
+Project Artemis is broken down into 4 key phases which will run sequentially, find more details of each below:
 
 ## Testnet Phases
 
-Project Artemis consists of 4 key phases:
+### Phase 1 - Apollo
 
-**Phase 1 - Apollo: Setup**
+Starter setup tasks for Cudos Network onboarding for Validators and users, focussing on account creation, hardware provisioning, node syncing and the CUDOS faucet
 
-Starter tasks for Cudos Network onboarding for Validators and users, focussing on account creation, hardware provisioning, node syncing and the CUDOS faucet
+#### Validator Tasks
 
-**Phase 2 - Buzz: Platform Testing**
+|**Task ID**|**Description**|**Resources**|**Reward**|
+| - | - | - | - |
+|`ARTEMIS-APOLLO#1`|Setup a Cudos Validator|[Docs](/build/#how-to-run-a-cudos-validator-node)|10|
+|`ARTEMIS-APOLLO#2`|Add a Sentry Node to your Validator|[Docs](/build/#run-a-sentry-node)|5|
+|`ARTEMIS-APOLLO#3`|Add Prometheus monitoring to the Validator Node|[Docs](https://prometheus.io/docs/prometheus/latest/getting_started/)|-|
+|`ARTEMIS-APOLLO#4`|Create a Grafana dashboard to show `ARTEMIS-APOLLO#3` Prometheus data/sending Validator data monthly to Cudo Network|[Docs](https://prometheus.io/docs/visualization/grafana/)|100|
+|`ARTEMIS-APOLLO#6`|Write a unique blog post setting up a Validator Node (in any language of your choice)|n/a|-|
+
+#### Developer Tasks
+
+|**Task ID**|**Description**|**Resources**|**Reward**|
+| - | - | - | - |
+|`ARTEMIS-APOLLO#5`|Install Keplr and get some tokens from the faucet using the explorer|[Docs](/build/account-setup.html)|10|
+|`ARTEMIS-APOLLO#7`|Join our community [Telegram](https://t.me/cudostelegram), [Discord](https://discord.com/invite/NUmUXEmvBn), and follow CUDOS on [Twitter](https://twitter.com/CUDOS_)|n/a|1|
+
```

Comment:
should we add the stage number? So that it's clear when it's updated for phase 2, or do you think there's no need?

Reply:
It's kind of a marketing decision on how we present this new breakdown of stages within phases, I think we stand a chance of just confusing users with this to be honest. 

How do you feel about us just updating this list in a weeks time with some new tasks and say something with the message of "Hey, we've added some bonus tasks for Phase 1, see them in bold here!", not worrying about trying to explain stages?

Reply:
yes that's fair enough, and then if we want we can explain the stages on phase 2

Reply:
Cool let's leave this as is then and let marketing know they'll be another batch of tasks to announce